hw/xilinx/ips/hpc/xlnx_ddr4/config.tcl

```diff
@@ -0,0 +1,20 @@
+# Author: Manuel Maddaluno <manuel.maddaluno@unina.it>
+# Description: DDR4 SDRAM (MIG) IP configuration file
+
+create_ip -name ddr4 -vendor xilinx.com -library ip -version 2.2 -module_name $::env(IP_NAME)
+
+set_property -dict [list \
+    CONFIG.System_Clock {Differential} \
+    CONFIG.C0.DDR4_AUTO_AP_COL_A3 {true} \
+    CONFIG.C0.DDR4_AxiAddressWidth {34} \
```

Comment:
Do we want config-based DDR4_AxiAddressWidth or just to max it out at 34?
Considering that 34 is max, RV64 config (ADDR_WIDTH=64) woud directly apply to this.
Maybe we should consider an address width converter.

Reply:
Sticking to 34 is fine imho

Reply:
I think 34 is good

Reply:
For a 32-bit we can just leave [33:32] = '0. For a 64-bit configurations we can just trim the higher bits. A more elegant and refined solution can be discussed, but it should not be the scope of this PR imho. (also because we do not support 64 yet)

Reply:
We leave hardcoded 32 for now